src/test/java/com/axibase/tsd/api/method/command/CompressedPayloadTest.java

```
@@ -0,0 +1,87 @@
+package com.axibase.tsd.api.method.command;
+
+import com.axibase.tsd.api.Checker;
+import com.axibase.tsd.api.method.checks.SeriesCheck;
+import com.axibase.tsd.api.method.extended.CommandMethod;
+import com.axibase.tsd.api.method.sql.SqlTest;
+import com.axibase.tsd.api.model.command.SeriesCommand;
+import com.axibase.tsd.api.model.series.Sample;
+import com.axibase.tsd.api.model.series.Series;
+import com.axibase.tsd.api.util.Mocks;
+import com.axibase.tsd.api.util.NotCheckedException;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.util.Collections;
+import java.util.zip.GZIPOutputStream;
+
+import static com.axibase.tsd.api.util.Mocks.entity;
+import static com.axibase.tsd.api.util.Mocks.metric;
+import static java.util.Collections.singletonMap;
+import static org.testng.Assert.assertTrue;
+
+public class CompressedPayloadTest extends SqlTest {
+    private static String outputGzipFilePath;
+
+    private static final String entityName = entity();
+    private static final String metricName = metric();
+
+    private static boolean checkResult(Series series) {
```

Do we really want to swallow exception? Maybe use exception message in assertion message?